shared/actions/favorite.js

```diff
@@ -79,6 +79,7 @@ const folderToProps = (folders: Array<Folder>, username: string = ''): FolderPro
     showingPrivate: true,
     smallMode: false,
     showComingSoon: !flags.tabFoldersEnabled,
+    username,
```

Comment:
username is already part of the store in the config tree so you can inject it as part of the smart component and remove it from here if you want to